### ISSUE TITLE
GetNote should also accept Notebook's name

### ISSUE THREAD

**Issue by y2k-shubham** [noculture/notes#15]
- just like `DeleteNote` that accepts both `notebookName` and `noteId`
 - `GetNote` must also accept `notebookName` in addition to `noteId`

```
         ..
	AddNote(notebookTitle string, note Note) error
	DeleteNote(notebookName string, noteId uint64) error
	GetNote(noteIndex uint64) (Note, error)
	// db-backup operation
        ..
```

[reference](https://github.com/noculture/notes/blob/master/models/db.go#L20)

**Comment by ifennna** [comment]
This works. It would be faster to search directly for a note in its bucket. I'm wondering whether this method is really necessary though, seeing as it isn't used anywhere.

**Comment by ifennna** [comment]
This issue has been addressed in PRs #24 and #28
